core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAO.scala

```diff
@@ -85,44 +87,62 @@ class BillingProfileManagerDAOImpl(samDAO: SamDAO,
       case Some(value) => value
     }
 
-    for {
-      billingProfiles <- getAllBillingProfiles(azureConfig, userInfo)
-    } yield {
-      billingProfiles.filter {
-        bp => samUserResources.map(_.resourceId).contains(bp.projectName.value)
-      }
-    }
-  }
-
-  private def getAllBillingProfiles(azureConfig: AzureConfig, userInfo: UserInfo)(implicit ec: ExecutionContext): Future[Seq[RawlsBillingProject]] = {
-    // NB until the BPM is live, we are returning a hardcoded
-    // Azure billing profile, with access enforced by SAM
+    // NB until the BPM is live, we want to ensure user is in the alpha group
     samDAO.userHasAction(
       SamResourceTypeNames.managedGroup,
       azureConfig.alphaFeatureGroup,
       SamResourceAction("use"),
       userInfo
     ).flatMap {
       case true =>
-        Future.successful(
-          Seq(
-            RawlsBillingProject(
-              RawlsBillingProjectName(azureConfig.billingProjectName),
-              CreationStatuses.Ready,
-              None,
-              None,
-              azureManagedAppCoordinates = Some(
-                AzureManagedAppCoordinates(
-                  UUID.fromString(azureConfig.azureTenantId),
-                  UUID.fromString(azureConfig.azureSubscriptionId),
-                  azureConfig.azureResourceGroupId
-                )
+        val profileApi = apiClientProvider.getProfileApi(userInfo.accessToken.token)
+        Future.successful(profileApi.listProfiles(0, 1000).getItems.asScala.toSeq)
+      case _ => Future.successful(Seq())
+    }
+  }
+
+  // Code copied from above, but method will be deleted.
```

Comment:
I have created https://broadworkbench.atlassian.net/browse/WOR-486 to delete all this code associated with the hardcoded account.